index.js

```diff
@@ -191,7 +190,19 @@ class Conf {
 		return false;
 	}
 
+	_isVersionInRangeFormat(version) {
```

Comment:
You should not have to check whether the version is in range format. Then the logic here is incorrect.

Reply:
`semver` requires different methods depending on whether we're using ranged or fixed versions.

`satisfies` only works with ranges, while `lte`, `gte`, etc, only works with fixed versions.
Do you suggest another approach to this? 😕

Reply:
It still feels weird having to use a regex to check for this. Seems like `semver` should solve that problem somehow. The regex is not correct either, it doesn't match versions like `1.1.1-beta`. You should probably use `semver.valid`.

Reply:
Yes, nice catch 👍

Reply:
Actually the most easy and straight-forward approach was to use `semver.clean` :)

Added a test to validate your use-case as well.